### PR TITLE
Adds experiments for 2.0

### LINKED ISSUE #20: example-get-started: long names in params file
The param names can be shortened to improve commands output for `dvc params diff` and especially `dvc exp show`.

Suggested names:
1. `featurize` group --> `fr`
2. `max_features`--> `max_fr`
3. `n_estimators`--> `n_est`

### PR DESCRIPTION
**Pull request by dberenbaum**
Updates to match https://github.com/iterative/dvc.org/pull/2195. Also fixes #20.

`deploy.sh` and `generate.sh` scripts ran with no errors and expected outputs.

### CODE REVIEW

File: example-get-started/code/params.yaml [hunk 1]
```diff
@@ -2,10 +2,11 @@ prepare:
   split: 0.20
   seed: 20170428
 
-featurize:
-  max_features: 500
+fr:
```
Comment (shcheklein) [inline example-get-started/code/params.yaml:5]: @dmpetrov @dberenbaum I know where it comes from, but don't you think it's better for the entry tutorial to be extra explicit and friendly? Does it hurt the layout too much with the longer names?

(eventually we'll need to figure out the UI to cover longer names anyway, right?)

Comment (dberenbaum) [reply example-get-started/code/params.yaml:5]: I knew I should have kept this separate 😆! I mentioned it to @dmpetrov and looked through the docs to see if it seemed hard to follow the parameters with the short names. I don't think it's too bad, but I can see both sides of it.

Comment (shcheklein) [reply example-get-started/code/params.yaml:5]: > I knew I should have kept this separate

yeah, since it'll force us to go and update docs in a lot of places (besides this discussion itself) :)

Comment (shcheklein) [reply example-get-started/code/params.yaml:5]: I'm totally fine if you discussed and agreed that it makes sense. Up to you Dave on this.

Comment (dberenbaum) [reply example-get-started/code/params.yaml]: I went with a compromise. The only place I notice a difference visually with the shorter names is with the train parameters that make `dvc exp show` output too wide, so only those parameters are shortened:

* `n_estimators` -> `n_est`
* `min_samples_split` -> `min_split`

This keeps the table width reasonable in the example without too much loss of clarity.

File: example-get-started/code/src/evaluate.py [hunk 2]
```diff
@@ -3,18 +3,18 @@
 import pickle
 import json
 
-from sklearn.metrics import precision_recall_curve
 import sklearn.metrics as metrics
 
-if len(sys.argv) != 5:
+if len(sys.argv) != 6:
     sys.stderr.write('Arguments error. Usage:\n')
     sys.stderr.write('\tpython evaluate.py model features scores plots\n')
```
Comment (shcheklein) [inline example-get-started/code/src/evaluate.py:10]: update usage?

File: example-get-started/generate.sh [hunk 3]
```diff
@@ -109,7 +110,8 @@ dvc run -n evaluate \
         -d src/evaluate.py -d model.pkl -d data/features \
         -M scores.json \
         --plots-no-cache prc.json \
-        python src/evaluate.py model.pkl data/features scores.json prc.json
+        --plots-no-cache roc.json \
+        python src/evaluate.py model.pkl data/features scores.json prc.json roc.json
 dvc plots modify prc.json -x recall -y precision
```
Comment (shcheklein) [inline example-get-started/generate.sh:115]: do we need to do `dvc plots modify` for the second plot?

File: example-get-started/generate.sh [hunk 4]
```diff
@@ -109,7 +110,8 @@ dvc run -n evaluate \
...
 git add .gitignore dvc.yaml dvc.lock prc.json scores.json
```
Comment (shcheklein) [inline example-get-started/generate.sh:116]: add roc.json?

**Comment by shcheklein** [comment]
we probably need to update src/README.md for the project a bit?

File: example-get-started/generate.sh (hunk 3 above)
Comment (dberenbaum) [reply example-get-started/generate.sh:115]: Yeah, and I need to update the docs here, too. Unfortunately, it adds a little complication in that it's no longer possible to plot everything with one line.

**Comment by dberenbaum** [comment]
> we probably need to update src/README.md for the project a bit?

Good point. I added the experiments section. Seems like the existing material is still pretty accurate.

**Review by shcheklein — APPROVED** [review]
LGTM! Awesome update, thanks 🙏 Please, feel free to squash-merge any time when you ready. Let me know if you'd like to update the `example-get-started` repo yourself or I can run the script for you

**Comment by dberenbaum** [comment]
Thanks! I don't mind running the script to update the remote and regenerate locally, but how do you update on GitHub? Do you just delete and recreate the repo? I don't think I have permissions on that repo either.

**Comment by shcheklein** [comment]
@dberenbaum thanks!

```
If the Github repo already exists, run these commands to rewrite it:

cd build/example-get-started
git remote add origin git@github.com:iterative/example-get-started.git
git push --force origin master
git push --force origin --tags
cd ../..
```

something like this

**Comment by dberenbaum** [comment]
Hah, I can't believe I never realized I could overwrite my repo with a force push!

I still don't have write access to https://github.com/iterative/example-get-started, so could you add me unless you want to do it yourself?

**Comment by shcheklein** [comment]
> I still don't have write access to https://github.com/iterative/example-get-started, 

@dberenbaum Oops, done!

**Comment by dberenbaum** [comment]
Weird, I'm still unable to push to https://github.com/iterative/example-get-started:

```
$ git push --force origin master
ERROR: Permission to iterative/example-get-started.git denied to dberenbaum.
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
```

Even trying to push to a new branch doesn't work, so I don't think it's a branch protection issue.

**Comment by shcheklein** [comment]
@dberenbaum sorry, I think I gave you access this repo, not the example-get-started itself. Please, give it a try again.

**Comment by dberenbaum** [comment]
Done. Thanks!
